node-operators/block-producers/operational-tasks/upgrade-your-node.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Upgrade your Node
+description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi Appchain protocol. 
+---
+
+# Upgrade your Node
+
+## Introduction {: #introduction }
+
+This tutorial covers upgrading your Tanssi block producer node. This guide assumes that you've already set up your account and spun up a [block producer node](/node-operators/block-producers/onboarding/run-a-block-producer){target=\_blank}. This guide will show you how to upgrade your node whether you're using [Docker](/node-operators/block-producers/onboarding/run-a-block-producer/block-producer-docker/){target=\_blank} or [Systemd](/node-operators/block-producers/onboarding/run-a-block-producer/block-producer-systemd/){target=\_blank}. 
+
+## Upgrading Docker Nodes {: #upgrading-docker-nodes }
+
+If you're running your block producer node with Docker, updating your node is as simple as stopping the running container and restarting it with the new version tag.
+
+First, get the container ID of your Tanssi block producer node with the following command: 
+
+```bash
+docker ps -a
+```
+
+The container ID is the first column and if you're running multiple docker containers, you can identify it by the image name of `moondancelabs/tanssi`. You can run the stop command as follows: 
+
+```bash
+docker stop INSERT_YOUR_CONTAINER_ID_HERE
+```
+
+Your terminal interaction will resemble the following:
+
+--8<-- 'code/node-operators/block-producers/operational-tasks/upgrade-your-node/terminal/docker-stop.md'
+
+To restart the node, you can use the command you used when launching your block producer node. The Docker command is configured to automatically retrieve the latest version.  
+
+=== "Generic"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+=== "Intel Skylake"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-skylake" \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+=== "AMD Zen3"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-znver3" \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
```

Comment:
Just to be explicit, it would be nice to state that the node will continue to sync where it last left off